django_prometheus/db/backends/postgis/base.py

```diff
@@ -0,0 +1,23 @@
+import psycopg2.extensions
+from django.contrib.gis.db.backends.postgis import base
+from django_prometheus.db.common import DatabaseWrapperMixin, ExportingCursorWrapper
+
+
+class DatabaseFeatures(base.DatabaseFeatures):
+    """Our database has the exact same features as the base one."""
+
+    pass
```

Comment:
this is not needed.
See https://help.semmle.com/wiki/display/PYTHON/Unnecessary+pass

Reply:
Got it. Why is it there in `mysql/base.py` and `postgresql/base.py`?

Reply:
Removed this block.